lib/apiml.js

```diff
@@ -177,25 +177,25 @@ ApimlConnector.prototype = {
         done(requestOpts);
       }
     }
-    log.debug("zluxProxyServerInstanceConfig: " 
-        + JSON.stringify(zluxProxyServerInstanceConfig, null, 2))
+    log.debug("ZWED0144I", JSON.stringify(zluxProxyServerInstanceConfig, null, 2)); //log.debug("zluxProxyServerInstanceConfig: " 
+        //+ JSON.stringify(zluxProxyServerInstanceConfig, null, 2))
     const url = `https://${this.apimlHost}:${this.apimlPort}/eureka/apps`
     zluxProxyServerInstanceConfig.eureka.serviceUrls = {
       'default': [
         url
       ]};
-    log.info(`Registering at ${url}...`);
-    log.debug(`zluxProxyServerInstanceConfig ${JSON.stringify(zluxProxyServerInstanceConfig)}`)
+      log.info(`ZWED0020I`, `${url}`); //log.info(`Registering at ${url}...`);
+      log.debug("ZWED0145I", `${JSON.stringify(zluxProxyServerInstanceConfig)}`); //log.debug(`zluxProxyServerInstanceConfig ${JSON.stringify(zluxProxyServerInstanceConfig)}`)
     const zluxServerEurekaClient = new eureka(zluxProxyServerInstanceConfig);
     //zluxServerEurekaClient.logger.level('debug');
     this.zluxServerEurekaClient = zluxServerEurekaClient;
     return new Promise(function (resolve, reject) {
       zluxServerEurekaClient.start(function (error) {
         if (error) {
-          log.warn(error);
+          log.warn('ZWED0005W', error); //log.warn(error);
```

Comment:
This needs to be tested to confirm if the behavior is equivalent to what it was before.
An important behavior in console logging:
If I have an error object, it has properties within such as message (string) or the stack trace function.
If I put error into a string, such as 'ZWED005W '+error, then it may show up as:
"ZWED005W [Error error]" or "ZWED005W the error message"
However, if I add it as an argument instead of appending it to the string, such as 'ZWED005W',error, then it may show up as:
"ZWED005W
Error:
message: the error message
stack trace: 
...
"
NodeJS and browsers essentially pretty-print objects if it can tell what type of object they are, as long as you dont request it in the string but instead as an argument.

Now, in our logging we have:
"ZWED0005W":"%s",

I am concerned this may have lost the error info by turning it into a string. If that is true, then the solution would be:
"ZWED0005W":"",
In other words, we're not adding it to the string but saving it as an argument at the end of processing.

It needs to be tested to see what the behavior is here, since this could effect many lines in this PR.

Reply:
OK, unfortunately, my suspicion was confirmed:
> let myError = new Error('Just a test');
undefined
> console.log('This is my error',myError);
This is my error Error: Just a test
    at repl:1:15
    at Script.runInThisContext (vm.js:122:20)
    at REPLServer.defaultEval (repl.js:332:29)
    at bound (domain.js:402:14)
    at REPLServer.runBound [as eval] (domain.js:415:12)
    at REPLServer.onLine (repl.js:642:10)
    at REPLServer.emit (events.js:203:15)
    at REPLServer.EventEmitter.emit (domain.js:448:20)
    at REPLServer.Interface._onLine (readline.js:308:10)
    at REPLServer.Interface._line (readline.js:656:8)
undefined
> console.log('This is my error'+myError);
This is my errorError: Just a test
undefined
> console.log('This is my error %s',myError);
This is my error Error: Just a test



So, basically everywhere that an object was being printed instead of string concatenated needs to be omitted from the %s scheme.
Most of this would occur at the end of a call, since you can't do this mid-string.
However you can have strings in between, so be careful of this example:
console.log('First object',object1,'Second object',object2);